android/src/main/java/com/openxchange/deltachatcore/DeltaChatCorePlugin.java

```diff
@@ -232,21 +227,6 @@ private void systemInfo(Result result) {
         result.success(android.os.Build.VERSION.RELEASE);
     }
 
-    private void coreListener(MethodCall methodCall, Result result) {
-        Boolean add = methodCall.argument(ARGUMENT_ADD);
-        Integer eventId = methodCall.argument(ARGUMENT_EVENT_ID);
-        if (eventId == null || add == null) {
-            return;
-        }
-        if (add) {
-            eventChannelHandler.addListener(eventId);
-            result.success(null);
-        } else {
-            eventChannelHandler.removeListener(eventId);
-            result.success(null);
-        }
-    }
-
```

Comment:
Wait, the entire `coreListener` stuff is gone? So we just have to take a lookup in an `DELEGATE_EVENTS` array if an incoming event is an element of this array? 🤔

Maybe we should discuss tomorrow (Thursday) face to face.